stage3/nodes.tf

```diff
@@ -73,7 +73,7 @@ module "ext_cardano_node" {
           effect   = "NoSchedule"
           key      = "kubernetes.io/arch"
           operator = "Equal"
-          value    = "amd64"
+          value    = "arm64"
```

Comment:
Do we want to hardcode a specific architecture here? We could have tolerations for both of them